app/helpers/db_flavor.py

```diff
@@ -17,3 +18,27 @@
         'port': 3306
     }
 }
+
+# Database flavours
+database_flavours = [
+    {
+        'name': 'mysql',
+        'host': os.getenv('ADMIN_MYSQL_HOST'),
+        'port': os.getenv('ADMIN_MYSQL_PORT'),
+        'class': MysqlDbService()
+    },
+    {
+        'name': 'postgres',
+        'host': os.getenv('ADMIN_PSQL_HOST'),
+        'port': os.getenv('ADMIN_PSQL_PORT'),
+        'class': PostgresqlDbService()
+    }
+]
+
+def get_db_flavour(flavour_name=None):
+    if flavour_name == 'mysql':
+        return database_flavours[0]
+    elif flavour_name == 'postgres':
```

Comment:
In this case we do not need the `if .. elif .. else`. I think just using 2 ifs is more idiomatic, more readable and easier to maintain. 
`
if flavour_name == 'mysql':
      return ...
if flavour_name == 'postgres':
      return ...
return False
`